src/components/analytics/Basic.vue

```diff
@@ -95,6 +102,9 @@ export default defineComponent({
         start_date: this.startDate,
         end_date: this.endDate,
       });
+    },
+    redirectToStockStatement() {
+      this.$router.push({path: '/analytics/stock'});
```

Comment:
Any reason for this function?

Reply:
I initially created a button to route to stock statement page and this function was linked to it but then i integrated stock statement with the tabs. so i will remove the function now